packages/react-sdk/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Webstudio <github@webstudio.is>",
   "homepage": "https://webstudio.is",
   "scripts": {
+    "dev": "tsup src/index.ts --format esm,cjs --watch --out-dir=lib",
```

Comment:
we already have the same thing in asset-uploader cc @SaraVieira added recently

Reply:
basically we need to add it in every package that is using ts, right?

Reply:
Will update this PR to do the same for the asset uploader and test